src/test/config/jira/service-expose-node.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: ${SERVICE_NAME}
+spec:
+  type: ClusterIP
+  ports:
+    - port: ${PORT_PRODUCT_SERVICE}
+      protocol: TCP
+      targetPort: ${PORT_PRODUCT_CONTAINER}
```

Comment:
The `targetPort` can instead be the name of a container port, so this can just be `http` (since that's the name we use for that container port in all 3 products, regardless of the numerical value)